rostime/include/ros/duration.h

```diff
@@ -120,6 +120,7 @@ class ROSTIME_DECL Duration : public DurationBase<Duration>
   explicit Duration(const Rate&);
   /**
    * \brief sleep for the amount of time specified by this Duration.  If a signal interrupts the sleep, resleeps for the time remaining.
+   * @return True if the desired sleep duration was met, false otherwise.
```

Comment:
No, because depending on what time it uses it might not be interrupted by g_stopped becoming true. And if it does I think it is covered by True if the desired sleep duration was met.